components/NavDrawer/index.jsx

```diff
@@ -183,11 +188,19 @@ class NavDrawer extends Component {
 
         {user && <Divider close={this.handleClose} />}
         {user && (
```

Comment:
Please add this check around the item instead of the entire block. It gives developers more freedom to choose what to do